### DISCUSSION TITLE
Dwin Firmware compatible creality touch screen

### DISCUSSION THREAD

**Discussion by pauluchin** [mriscoc/Ender3V2S1#48] [Q&A] [answered]
Hello everyone... I have a question... is the Dwin firmware compatible with the new creality touch screen???

Hola a todos... tengo una consulta.. el firmware Dwin es compatible con la nueva pantalla tactil de creality ???

**Comment by mriscoc** [comment, marked as answer]
Currently there is no support for display units with touch screen.
